graylog2-web-interface/src/components/common/ExpandableListItem.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox } from 'react-bootstrap';
+
+import style from './ExpandableListItem.css';
+
+/**
+ * The ExpandableListItem is needed to render a ExpandableList.
+ */
+class ExpandableListItem extends React.Component {
+  static propTypes = {
+    /** Is the Item checked */
+    checked: PropTypes.bool,
+    /** TODO: i do not know what this means */
```

Comment:
Now that I think of it, the name is actually not correct, it should be called `indetermined` instead. I would document it as:

>Indicates whether the checkbox on this item should be in an indetermined state or not. This is mostly helpful to represent cases where the element is only partially checked, for instance when `ExpandableListItem`'s child is an `ExpandableList` and some of its items are checked, but others are not.